03LessonThree/LessonThree.js

```diff
@@ -2,33 +2,36 @@ module.exports = function() {
 
   this.buildArray = function (first, second, third) {
     // return array with the items first, second, and third in an array.
-
+    var array = [];
+    array.push(first, second, third);
```

Comment:
Done.

Thanks for your feedback Joe! Let me know if you have trouble seeing the changes.